QUARKUS-710.md

```diff
@@ -0,0 +1,56 @@
+# QUARKUS-710 - Add Support for Micrometer
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-710
+
+Micrometer is a metrics instrumentation library for JVM-based applications. It provides a simple facade over the instrumentation clients for the most popular monitoring systems. Currently, the testing will only scope the integration with Prometheus.
+
+For native mode similar test coverage as for JVM mode needs to be ensured.
+
+## Scope of the testing
+
+Following actions were taken to ensure test coverage and automation for native image:
+ - Define the scope of the testing
+ - Analyze the possible impact on Quarkus itself - e.g performance, compatibility
+ - Determine impact on test suites - e.g. new test suite, increased complexity of existing test suite, maintenance burden
+ - Determine impact on QE infrastructure - e.g. increased complexity of existing automation, maintenance burden, need for new external resources
+
+### Impact on testsuites and testing automation:
+ - OpenShift TS framework needs to be enhanced to install Prometheus
+ - Smoke tests to be added into the OpenShift TS:
+   - Verify expected metrics are shared between the Quarkus application and Prometheus
+ - More test development needs to be added in Beefy-scenarios testsuite:
+   - Scenario with parallel processing of metrics
+   - Scenario with custom tags using `@MeterFilterConstraint` annotation
+   - Scenario to use a custom Prometheus registry
+   - Support for the MicroProfile Metrics API
+   - Scenarios to cover the correct functionality of the following properties:
```

Comment:
There are quite a few bullet-points. Which of them are mandatory and which can be treated as optional?

Reply:
All the configuration bullets are optional, so if I understood you correctly, I will remove them.